### ISSUE TITLE
web admin: unable to restart a service

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#6946]
**Describe the bug**
If you want to restart a service using Services menu on web admin, you can't.

There is an issue with buttons.

**To Reproduce**
1. Go to Status -> Services menu
2. Try to restart a service (for example: haproxy-portal)

=> When you switch state using "Running" button, service is stopped but the toggle doesn't switch to left. Consequently, you can't start service because toggle is already on the right

**Screenshot**
Screenshot after I stopped haproxy-portal service:
![image](https://user-images.githubusercontent.com/10830260/159641438-4fcc36f5-ef80-411c-a3ae-f33470e6239c.png)

**Expected behavior**
When you stop service, toggle should move to left.

**Comment by nqb** [comment]
I'm also assigning @jrouzierinverse because I noticed that API returns is bad.

After I stop `httpd.aaa`, API response to `/status` is:
```json
{
	"alive": 1,
	"enabled": 1,
	"id": "httpd.aaa",
	"managed": 1,
	"pid": "0",
	"status": 200
}
```

Certainly related to #4817.

It's on Debian 11

**Comment by jrouzierinverse** [comment]
This has been fixed by 808d7859afd059ca50c8725c34b825bdfce4966f
